### ISSUE TITLE
Creating <object>

### ISSUE THREAD

**Issue by ip1981** [chrisdone/lucid#72]
It seems I can't get such HTML, because `data_` is not exactly what I need:
```html
<object data="movie.swf"
  type="application/x-shockwave-flash"></object>
```
Existing `data_` takes two arguments and yields `data-foo="bar"`.

Please advise :) Maybe raw HTML is an option.

**Comment by chrisdone** [comment]
Good point. I should probably add a helper for this. You can use `makeAttribute "data" "movie.swf"` for now, yeah.

**Comment by chrisdone** [comment]
Trying to think of a name for this, I looked at the docs for the `data` attribute:

> `data`
> The address of the resource as a valid URL. At least one of data and type must be defined.

So perhaps we can go with `dataUrl_`.
